### PR TITLE
Adopt erc standards

### PR DESCRIPTION
**Pull request by darrenvechain**

### CODE REVIEW

**Review by libotony — APPROVED** [review]

File: vips/VIP-250.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+Title: ERC Contract Adoption
```
Comment (NecoSherry) [inline vips/VIP-250.md:2]:
```suggestion
VIP: 241
Title: ERC Contract Adoption
```

File: vips/VIP-250.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
```
Comment (NecoSherry) [inline vips/VIP-250.md:1]: Rename the file as 'VIP-241.md'

File: the-status-page.md [hunk 3]
```diff
@@ -21,10 +19,10 @@ All Vechain Improvement Proposals
...
 | [200](./vips/VIP-200.md) | Implementation of SURFACE BFT Protocol | Ziheng Zhou (peter.zhou@vechain.com) and Zhijie Ren (zhijie.ren@vechain.com) | Core | Draft | - |
 | [201](./vips/VIP-201.md) | Simple Gas Payer Standard | Xiqing Chu <xiqing.chu@vechain.com> & Mog Lu <mog.lu@vechain.com> | Information | Draft | - |
-| [210](./vips/VIP-210.md) | Semi-Fungible Token(SFT) Standard | VeChain | Application | Draft | - |
 | [214](./vips/VIP-214.md) | Per Epoch VRF-Based Random Beacon Generation | Zhijie Ren (zhijie.ren@vechain.com) and Ziheng Zhou (peter.zhou@vechain.com) | Core | Draft | - |
 | [220](./vips/VIP-220.md) | Finality with One Bit (FOB) | Zhijie Ren (zhijie.ren@vechain.com) and Ziheng Zhou (peter.zhou@vechain.com) | Core | Draft | - |
 | [240](./vips/VIP-240.md) | VET Custodian Contract Standard | Xiqing Chu (xiqing.chu@vechain.org) | Application | Draft | [Peter Zhou](https://github.com/zzGHzz), [Bin Qian](https://github.com/qianbin), [Darren Kelly](https://github.com/darrenvechain) |
+| [250](./vips/VIP-250.md) | ERC Contract Adoption | Darren Kelly (darren.kelly@vechain.org) | Application | Draft
```
Comment (NecoSherry) [inline the-status-page.md:25]:
```suggestion
| [241](./vips/VIP-241.md) | ERC Contract Adoption | Darren Kelly (darren.kelly@vechain.org) | Application | Draft
```

Comment (darrenvechain) [reply the-status-page.md]: Resolved in latest commit

File: vips/VIP-250.md (hunk 1 above)
Comment (darrenvechain) [reply vips/VIP-250.md]: Resolved in latest commit

**Review by daithihearn — APPROVED** [review]
LGTM

**Comment by darrenvechain** [comment]
@NecoSherry should be ready now I think

**Review by NecoSherry — APPROVED** [review]

**Pull request merged**